problems/strings.js

```diff
@@ -105,7 +147,15 @@ function everyOtherLetter() {}
  * example: disemvowel("aeiou") returns ""
  * example: disemvowel("Hello world!") returns "Hll wrld!"
  */
-function disemvowel() {}
+function disemvowel(str) {
+let newStr = ""
+for (let i = 0; i <=str.length-1;i++){
+  if(str[i]!=='a' && str[i]!=='e' && str[i]!=='i' && str[i]!=='o' && str[i]!=='u' && str[i]!=='A' && str[i]!=='E' && str[i]!=='I' && str[i]!=='O' && str[i]!=='U'){
```

Comment:
Anytime you have a long chain of comparisons like this, we call it a "code smell" which means there's likely a better way you could do this.

In this case, you could use .includes()

```js
let letters = 'aeiouAEIOU'
if(letters.includes(str[i]) {}
```